### PR TITLE
use a post deploy hook that will pass, not fail

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@php-coder ptal

File: examples/sample-app/application-template-stibuild.json [hunk 1]
```diff
@@ -356,7 +356,7 @@
               "failurePolicy": "Ignore",
               "execNewPod": {
                 "command": [
-                  "/bin/false"
+                  "/bin/true"
```
Comment (php-coder) [inline examples/sample-app/application-template-stibuild.json]: Hm.. and why we had used `/bin/false` before?

**Comment by php-coder** [comment]
LGTM.

File: examples/sample-app/application-template-stibuild.json (hunk 1 above)
Comment (bparees) [inline examples/sample-app/application-template-stibuild.json]: no idea. guessing someone was trying to test the behavior of failed post deploy hooks.

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
flake https://github.com/openshift/origin/issues/10080
[test]

**Comment by bparees** [comment]
flake https://github.com/openshift/origin/issues/9490
[test]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**